File: data/imported/knowledge/slides/java/merged/snippets/snippet_024.java
@Test
    void testRemoveCustomProperty() {
        try (Presentation pres = new Presentation()) {
            IDocumentProperties props = pres.getDocumentProperties();
            props.setCustomPropertyValue("A", "val");
            props.setCustomPropertyValue("B", "val");
            assertThat(props.getCountOfCustomProperties()).isEqualTo(2);

            props.removeCustomProperty("A");
            assertThat(props.getCountOfCustomProperties()).isEqualTo(1);
            assertThat(props.containsCustomProperty("A")).isFalse();
            assertThat(props.containsCustomProperty("B")).isTrue();
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_024.cs
[Fact]
    public void TestRemoveCustomProperty()
    {
        // Removing a custom property decreases count.
        using var pres = new Presentation();
        var props = pres.DocumentProperties;
        props.SetCustomPropertyValue("A", "val");
        props.SetCustomPropertyValue("B", "val");
        props.CountOfCustomProperties.Should().Be(2);

        props.RemoveCustomProperty("A");
        props.CountOfCustomProperties.Should().Be(1);
        props.ContainsCustomProperty("A").Should().BeFalse();
        props.ContainsCustomProperty("B").Should().BeTrue();
    }